commands/displayers/droplet_backup_policies.go

```diff
@@ -0,0 +1,50 @@
+package displayers
+
+import (
+	"io"
+
+	"github.com/digitalocean/doctl/do"
+)
+
+type DropletBackupPolicy struct {
+	DropletBackupPolicies []do.DropletBackupPolicy
+}
+
+var _ Displayable = &DropletBackupPolicy{}
+
+func (d *DropletBackupPolicy) JSON(out io.Writer) error {
+	return writeJSON(d.DropletBackupPolicies, out)
+}
+
+func (d *DropletBackupPolicy) Cols() []string {
+	cols := []string{
+		"DropletID", "BackupEnabled", "BackupPolicyPlan", "BackupPolicyWeekday", "BackupPolicyHour",
+		"BackupPolicyWindowLengthHours", "BackupPolicyRetentionPeriodDays",
+		"NextBackupWindowStart", "NextBackupWindowEnd",
+	}
+	return cols
+}
+
+func (d *DropletBackupPolicy) ColMap() map[string]string {
+	return map[string]string{
+		"DropletID": "Droplet ID", "BackupEnabled": "Backup Enabled",
+		"BackupPolicyPlan": "Backup Policy Plan", "BackupPolicyWeekday": "Backup Policy Weekday", "BackupPolicyHour": "Backup Policy Hour",
+		"BackupPolicyWindowLengthHours": "Backup Policy Window Length Hours", "BackupPolicyRetentionPeriodDays": "Backup Policy Retention Period Days",
+		"NextBackupWindowStart": "Next Backup Window Start", "NextBackupWindowEnd": "Next Backup Window End",
+	}
+}
+
+func (d *DropletBackupPolicy) KV() []map[string]any {
+	out := make([]map[string]any, 0)
+	for _, d := range d.DropletBackupPolicies {
```

Comment:
Lets not redefine `d` here for clarity.

```suggestion
func (d *DropletBackupPolicy) KV() []map[string]any {
	out := make([]map[string]any, 0)
	for _, policy := range d.DropletBackupPolicies {
```

Reply:
Done in [bc4ede2](https://github.com/digitalocean/doctl/pull/1609/commits/bc4ede23d4ca84d1ce6e2cd3361c0cbb095f24c7)